### PR TITLE
test: add event checking to TestMigrateContract

### PR DESCRIPTION
**Pull request by da1suk8**
### TestInstantiateContract
https://github.com/Finschia/wasmd/blob/main/docs/proto/proto-docs.md#msgmigratecontract

This PR adds an integration test for TestMigrateContract.

This test is based on https://github.com/CosmWasm/wasmd/blob/33df4aeaaa419f6aad07b94d64c0423bee0a2d21/x/wasm/keeper/msg_server_integration_test.go#L740-L825

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/Finschia/wasmd/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/wasmd/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 * [\#64](https://github.com/Finschia/wasmd/pull/64) test: add test cases to confirm output for PinnedCodes
 * [\#70](https://github.com/Finschia/wasmd/pull/70) add event checking to TestInstantiateContract
 * [\#73](https://github.com/Finschia/wasmd/pull/73) test: add the check for expPaginationTotal
-
+* [\#78](https://github.com/Finschia/wasmd/pull/78) test: add the check for TestMigrateContract
```
Comment (170210) [inline CHANGELOG.md:15]:
```suggestion
* [\#78](https://github.com/Finschia/wasmd/pull/78) test: add the check for TestMigrateContract

```

**Review by 170210 — DISMISSED** [review]
LGTM

File: x/wasm/keeper/msg_server_integration_test.go [hunk 2]
```diff
@@ -141,3 +146,104 @@ func TestInstantiateContract(t *testing.T) {
...
+				CodeID: storeCodeResponse.CodeID,
+				Label:  "test",
+				Msg:    initMsgBz,
+				Funds:  sdk.Coins{},
+			}
+			rsp, err = wasmApp.MsgServiceRouter().Handler(msgInstantiate)(xCtx, msgInstantiate)
+			require.NoError(t, err)
+			var instantiateResponse types.MsgInstantiateContractResponse
+			require.NoError(t, wasmApp.AppCodec().Unmarshal(rsp.Data, &instantiateResponse))
+			contract_addr := rsp.Data
```
Comment (170210) [inline x/wasm/keeper/msg_server_integration_test.go:208]: There seem to be some unexpected characters at the beginning of rsp.Data
```suggestion
			contract_addr := instantiateResponse.Address
```

File: x/wasm/keeper/msg_server_integration_test.go [hunk 3]
```diff
@@ -141,3 +146,104 @@ func TestInstantiateContract(t *testing.T) {
...
+			assert.Equal(t, "module", string(events[0].Attributes[0].Key))
+			assert.Equal(t, "wasm", string(events[0].Attributes[0].Value))
+			assert.Equal(t, "sender", string(events[0].Attributes[1].Key))
+			assert.Equal(t, myAddress.String(), string(events[0].Attributes[1].Value))
+			assert.Equal(t, "migrate", events[1].Type)
+			assert.Equal(t, 2, len(events[1].Attributes))
+			assert.Equal(t, "code_id", string(events[1].Attributes[0].Key))
+			assert.Equal(t, "1", string(events[1].Attributes[0].Value))
+			assert.Equal(t, "_contract_address", string(events[1].Attributes[1].Key))
+			assert.Contains(t, string(contract_addr), string(events[1].Attributes[1].Value))
```
Comment (170210) [inline x/wasm/keeper/msg_server_integration_test.go:244]:
```suggestion
			assert.Equal(t, string(contract_addr), string(events[1].Attributes[1].Value))
```

**Comment by da1suk8** [comment]
@170210 

I deleted `contract_addr` and I used `instantiate.Address` instead of `contract_addr`. 
Thanks.

**Review by 170210 — DISMISSED** [review]

**Review by 170210 — APPROVED** [review]

**Pull request merged**